packages/blocks/src/page-block/page-service.ts

```diff
@@ -233,4 +236,35 @@ export class PageService extends BlockService<PageBlockModel> {
   loadFonts() {
     this.fontLoader.load(CanvasTextFonts);
   }
+
+  private _getLastNoteBlock() {
+    const { page } = this;
+    let note: NoteBlockModel | null = null;
+    if (!page.root) return null;
+    const { children } = page.root;
+    for (let i = children.length - 1; i >= 0; i--) {
+      const child = children[i];
+      if (
+        matchFlavours(child, ['affine:note']) &&
+        child.displayMode !== NoteDisplayMode.EdgelessOnly
+      ) {
+        note = child as NoteBlockModel;
+        break;
+      }
+    }
+    if (!note) {
+      const noteId = page.addBlock('affine:note', {}, page.root?.id);
+      note = page.getBlockById(noteId) as NoteBlockModel;
+    }
+    return note;
+  }
+
+  appendParagraph = () => {
+    const note = this._getLastNoteBlock();
+    if (!note) return;
+    const id = this.page.addBlock('affine:paragraph', {}, note);
+    const host = document.querySelector('editor-host');
```

Comment:
pls don't use global queries... this will fail in case of multiple editors

Reply:
so how we get editor host in service?

Reply:
@regischen `this.std.host`

Reply:
okay, ty!